06_modules_lab/01.py

```diff
@@ -1,4 +1,14 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+
+import sys
+
+if len(sys.argv) < 2:
+	print "Error, please send number as argument"
+	sys.exit()

```

Comment:
Great! But it's better to write the error message inside sys.exit - such as `sys.exit("Error...")`

There are two reasons for this:
1. The output is written to the user as standard error (you won't always see the difference, but in some situations you will).
2. sys.exit() returns an exit code to the operating system telling if the script finished successfully or failed. Calling it with no arguments signals a success, but passing an error message signals to the OS that the script failed.

Both are not always visible except in situations a user actively queries them, but for users who do care it'd give a more correct script.